tool_wrapper.py

```diff
@@ -89,12 +89,17 @@ def main():
         os.path.basename(args.tool),
     ] + args.flag + [args.input]
 
-    result = subprocess.check_output(
-        args=cmd_args,
-        env=env,
-        stderr=subprocess.STDOUT,
-    )
-    
+    result = ""
+    try:
+        result = subprocess.check_output(
+            args=cmd_args,
+            env=env,
+            stderr=subprocess.STDOUT,
+            universal_newlines=True,
+        )
+    except subprocess.CalledProcessError as exc:
+        print("Failed to run tool: ", exc.returncode, exc.output)
```

Comment:
LGTM

But maybe for consistency.

Would it make sense to add the print out to the sys exit below?
With add an additional condition for success of the above call to the if-statement.

Or do we expect there to exist failures that we want to print but still generates the output?

Reply:
Good point! Done. I'm also checking the return code, so even if somehow an output is produced but a return code != 0 is returned, this should fail in the same way.